src/applications/mhv/secure-messaging/components/MessageList/FolderHeader.jsx

```diff
@@ -57,12 +57,13 @@ const FolderHeader = props => {
       <>{handleFolderDescription()}</>
       {folder.folderId === Folders.INBOX.id && <ComposeMessageButton />}
       <ManageFolderButtons />
-      {folder.count > 0 && (
+      {threadCount?.length > 0 && (

```

Comment:
I really think we should capture this logic in a unit test

Reply:
Added tests for this in this PR --> https://github.com/department-of-veterans-affairs/vets-website/pull/24991

Reply:
@vsaleem I am afraid I can't see the coverage for a filter box in that PR referenced above. Perhaps a commit is still pending to be pushed?

Reply:
@oleksii-morgun-va, The filterBox coverage is within this PR.. my mistake!